OIPA/iati/models.py

```diff
@@ -53,6 +53,7 @@ class Activity(models.Model):
     hierarchy = models.SmallIntegerField(choices=hierarchy_choices, default=1, null=True)
     # last_updated_datetime = models.CharField(max_length=100, default="")
     # value_date = models.DateField(null=True)
+    last_updated = models.DateTimeField(null=True, auto_now=True)
```

Comment:
This is a duplicate of last_updated_datetime? @daanzap or is this used for last update search indexes? if so, could this have a more descriptive name? 

@bryanph rest of the PR looks fine to me, could you also check it?